exercises/dominoes/.meta/src/reference/java/Dominoes.java

```diff
@@ -0,0 +1,75 @@
+import java.util.ArrayList;
+import java.util.Arrays;
+
+class Dominoes {
+	
+	public static void main(String[] args) {
+		Domino[] dominoesList = {new Domino(1, 2), new Domino(3, 1), new Domino(2, 3)};
+		ArrayList<Domino> dominoes = new ArrayList<>(Arrays.asList(dominoesList));
+		
+		ArrayList<Domino> chain = formPartialChain(new ArrayList<Domino>(), dominoes);
+		
+		for(int i = 0; i < chain.size(); i++) {
```

Comment:
Please add whitespace after each `for` and each `if`, e.g. 
```
for (int i = 0; i < chain.size(); i++) {
```
to be consistent with the other code in this repo 🙂

Reply:
Ok. I'm sorry I keep doing this. I guess old habits die hard.

Reply:
That's fine, like you said it's hard to change habits like this 🙂 And it's easy to miss when you're focusing on the code logic 🙂